pkg/plugins/resources/xml/target.go

```diff
@@ -32,6 +32,7 @@ func (x *XML) TargetFromSCM(source string, scm scm.ScmHandler, dryRun bool) (cha
 
 	if scm != nil {
 		x.spec.File = joinPathWithWorkingDirectoryPath(x.spec.File, scm.GetDirectory())
+		logrus.Debugf("Relative path detected: changing to absolute path from SCM: %q", x.spec.File)
```

Comment:
This change is not related to this PR: wrong rebase?

Reply:
Same as above.